crates/node/src/chain_spec.rs

```diff
@@ -85,9 +88,27 @@ pub fn development_config(enable_manual_seal: Option<bool>, madara_path: PathBuf
     ))
 }
 
+// helper to print development accounts info
+// accounts with addresses 0x1 and 0x4 are NO VALIDATE accounts (don't require PK)
+// accounts with addresses 0x2 and 0x3 have the same PK
+pub fn print_development_accounts(genesis_loader: GenesisLoader) {
+    let genesis_config: madara_runtime::pallet_starknet::GenesisConfig<Runtime> = genesis_loader.into();
```

Comment:
Why do you have to convert to GenesisConfig. The data is the same in GenesisLoader. Only the type changes. But for display it shouldn't be an issue